docs/src/maintainer-guide/pullrequests.md

```diff
@@ -41,7 +41,13 @@ Once the bot checks have been satisfied, you check the following:
 
 ## Who Can Merge a Pull Request
 
-TSC members and committers may merge pull requests, depending on the contents of the pull request.
+TSC members, Reviewers, Committers, and Website Team Members may merge pull requests, depending on the contents of the pull request.
+
+Website Team Members may merge a pull request in the `eslint.org` repository or the `docs` directory in the `eslint` directory if it is:
```

Comment:
Will the Website Team Members need push access to `eslint/eslint` repository as well? (per the governance document, "Website Team Members are given push access to the `eslint.org` GitHub repository" only).

Reply:
For now I was thinking no because we can’t limit them to just the docs directory. We can revisit later if we feel this is too restrictive.

Reply:
I agree, but then they won't be able to merge any pull requests in the `eslint/eslint` repo (which is in my understanding the second part of this sentence).

Reply:
Oops, good catch.